tests/main/security-devpts/pts.exp

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/expect
+
+set timeout 2
+
+send "ls /dev/pts"
+expect "\[0-9\]*ptmx"
```

Comment:
I haven't used expect for a while, so I took some time to get up to speed again and see that your script does not exit non-zero if I adjust `expect "\[0-9\]*ptmx"` to be `expect "nonexistent"`. You also aren't sending the ls command to anything useful, like bash. I think you want this instead:
```
spawn bash
send "ls /dev/pts\n"
expect "\[0-9]*ptmx" {} timeout { exit 1 }
send "exit\n"
```

```
expect {